tests/TestCase/View/CellTest.php

```diff
@@ -133,6 +133,44 @@ public function testDefaultCellAction()
     }
 
     /**
+     * Tests that cell action setting the template using the property renders the correct template
+     *
+     * @return void
+     */
+    public function testSettingCellTemplateFromAction()
+    {
+        $appCell = $this->View->cell('Articles::customTemplate');
+
+        $this->assertContains('This is the alternate template', "{$appCell}");
+        $this->assertEquals('alternate_teaser_list', $appCell->template);
+        $this->assertEquals('alternate_teaser_list', $appCell->viewBuilder()->template());
+
+        $appCell = $this->View->cell('Articles::customTemplate');
+
+        $this->assertContains('This is the alternate template', $appCell->render());
+        $this->assertEquals('alternate_teaser_list', $appCell->template);
+        $this->assertEquals('alternate_teaser_list', $appCell->viewBuilder()->template());
```

Comment:
No there should only be one test, this is a mistake.
I'll change it tonight.

Same thing goes for the test below.